pages/api/popular.ts

```diff
@@ -1,45 +1,55 @@
 import type { NextApiHandler } from "next"
 import { getToken } from "next-auth/jwt"
+import xss from "xss"
 
-import { config } from "config"
-import { HttpClient } from "lib/HttpClient"
 import { HttpError } from "lib/HttpError"
+import { Asset } from "model/Asset"
+import { makeCollibraService } from "services"
+import { getAssetAttributes } from "services/collibra"
 
-type PopularAsset = Collibra.Asset & Pick<Collibra.NavigationStatistic, "numberOfViews">
+type PopularAsset = Asset & Pick<Collibra.NavigationStatistic, "numberOfViews">
 
 const getPopularAssets = async (
   data: PopularAsset[],
-  authorization: string,
+  r: ReturnType<typeof makeCollibraService>,
```

Comment:
I find the `r` a little bit hard to read

Reply:
the type, or the parameter name?

Reply:
got you. i'll update it